bin/gh_pr_bootstrap.sh

```diff
@@ -92,7 +92,7 @@ function setup_cmsbot() {
     if [ ! -d ${CMS_BOT_DIR} ]; then
         (
             cd "$WORKSPACE"
-            git clone -b master git@github.com:Mu2e/CI
+            git clone -b main git@github.com:Mu2e/CI
         )
     else
         (
```

Comment:
In case there are workspaces / old Mu2e/CI directories from older jenkins builds, here it might be a good idea to add an extra check to make sure it's on the right branch

```bash
if [[ "$(git rev-parse --abbrev-ref HEAD)" == *"master"* ]]; then
  git fetch
  git checkout main
fi
```

Reply:
Yeah, that's a good point. Will update